src/components/Tree/Node.tsx

```diff
@@ -190,19 +190,17 @@ export const Node = ({
                             </span>
                         </div>
                     </a>
-                    <AnimatePresence>
-                        {actions && (isHovered || selected) && (
-                            <motion.div
-                                className="tw-flex tw-space-x-1.5 tw-items-center"
-                                initial={{ width: 0 }}
-                                animate={{ width: 'auto' }}
-                                exit={{ width: 0 }}
-                                transition={{ ease: [0.04, 0.62, 0.23, 0.98] }}
-                            >
-                                {actions.map((action) => action)}
-                            </motion.div>
-                        )}
-                    </AnimatePresence>
+
+                    {actions && (
+                        <div
+                            className={merge([
+                                'tw-flex tw-items-center',
+                                isHovered || selected ? 'tw-visible tw-space-x-1.5' : 'tw-invisible tw-w-0',
```

Comment:
What would be the purpose of having the visibility class? As the width is set on hover, I think it exactly the same as having `actions && (isHovered || selected) &&` on line 194. And that ternary condition might not be necessary. Don't you think?

Reply:
It's an accesibility condition. Hover is not available to all users, it's only the visibility that we want to set.

It also ensures that the height of the row doesn't change when showing/hiding the button.

Either visibility or overflow: hidden would probably be fine but I went with visibility as it's more semantically clear.

Reply:
I see! Got it, thanks for the explanation. Then that would be just removing the import and I can approve. 😄